Checkbox/themes/Checkbox_template.less

```diff
@@ -1,56 +1,55 @@
+label[for] {
+	// fixes a ios bug that makes it so that the handler set on document, 
+	// listening for clicks on label, is never called.
+	cursor: pointer;
+}
+
 .d-checkbox {
 	position: relative;
-	.d-checkbox-styles();
-	&.d-focused { // from tab-focus()
-		.d-checkbox-focused();
+	display: inline-block;
+	/* #408 Enforce content-box to ensure correctly centered check mark */
+	/* even if border-box is used everywhere else for instance */
+	/* via delite/themes/defaultapp.css */
+	box-sizing: content-box;
```

Comment:
Enforcing box-sizing looks obsolete with this refactoring. Same comment for radiobutton.